package.json

```diff
@@ -7,8 +7,8 @@
     "JSON2": "0.1.0",
     "batch": "0.5.0",
     "bouncy": "3.2.2",
-    "browserify": "6.3.3",
-    "browserify-istanbul": "0.1.3",
+    "browserify": "8.1.3",
```

Comment:
bumping browserify or other deps should be a separate commit unless this is required to make watchify work